Tests/SwiftFormatRulesTests/XCTestManifests.swift

```diff
@@ -42,7 +42,7 @@ extension BeginDocumentationCommentWithOneLineSummaryTests {
     //   `swift test --generate-linuxmain`
     // to regenerate.
     static let __allTests__BeginDocumentationCommentWithOneLineSummaryTests = [
-        ("testApproximationsOnMacOS", testApproximationsOnMacOS),
+        // ("testApproximationsOnMacOS", testApproximationsOnMacOS),
```

Comment:
Chatted offline; recommend instead of commenting this out (since it'll get blown away any time we run the generator again), update the test to move the `#if os(macOS)` check inside the method body instead of outside. That will make the test still exist on Linux, but as a no-op that always succeeds.

Reply:
Done.